docs/_instructions/maven-release.md

```diff
@@ -15,17 +15,21 @@ Though this instruction is not specific for a plugin, it was developed in conjun
     option         ::= sources | javadoc | pom | sign
     sources        ::= 'sources' 
                        ( ';path=' ( 'NONE' | PATH ) )?
+                       ( ';local=' ( 'true' | 'false' ) )?
                        ( ';-sourcepath=' PATH ( ',' PATH )* )?
     javadoc        ::= 'javadoc'
                        ( ';path=' ( 'NONE' | PATH ) )?
                        ( ';packages=' ( 'EXPORTS' | 'ALL' ) )?
+                       ( ';local=' ( 'true' | 'false' ) )?
                        ( ';' javadoc-option )*
     javadoc-option ::= '-' NAME '=' VALUE
     pom            ::= 'pom'
                        ( ';path=' ( 'JAR' | PATH ) )?
     sign            ::= 'sign'
                        ( ';passphrase=' VALUE )?
 
+If `sources` or `javadoc` has the attribute local set to true, aither one will be release to the maven repository even if no `releaseUrl` or `snapshotUrl` is set.
```

Comment:
`aither` is a misspelling. Also, I think you mean to say `local=true` will release to the local (e.g. `maven.repo.local` aka `~/.m2/repository`) repository.

Reply:
If decided to rename the attribute to force, because this is what it actually does. Force the deployment. I hope I haven't fucked anything up this time.